tsdb/index/tsi1/index_test.go

```diff
@@ -394,3 +338,11 @@ func (idx *Index) CreateSeriesSliceIfNotExists(a []Series) error {
 	}
 	return nil
 }
+
+func BytesToStrings(a [][]byte) []string {
```

Comment:
I see this in a couple places, would it make sense to add to the `bytesutils` package?

Reply:
Sounds good. I did consider this, but was concerned others' might object to putting code only used in tests into non-test packages.